src/core/Link/SkipLink/SkipLink.baseStyles.ts

```diff
@@ -13,7 +13,6 @@ export const SkipLinkStyles = (theme: SuomifiTheme) => css`
     background: ${theme.colors.highlightLight3};
     border: 1px solid ${theme.colors.depthLight1};
     color: ${theme.colors.blackBase};
-    text-decoration: none;
```

Comment:
Is this a necessary removal, since the skip link is quite a unique implementation in general? Not sure if the underline really brings any extra value here.

Reply:
This actually doesn't remove the underline by default, since SkipLink gets the styles from the BaseLink. So this style was redundant here.

Reply:
Ah, so it seems. Good removal in that case 😁